core/shared/src/main/scala/clue/ApolloClient.scala

```diff
@@ -643,7 +657,8 @@ class ApolloClient[F[_], S, CP, CE](
           val stream = bracket.flatMap(_ =>
             (
               Stream.eval(sendStart) >>
-                emitter.queue.dequeue
+                Stream
```

Comment:
`Queue` is not in fs2 anymore and not a stream. The method `fromQueueUnterminated` seems to do the conversion